lib/exercism_config/determine_environment.rb

```diff
@@ -0,0 +1,16 @@
+module ExercismConfig
+  class DetermineEnvironment
+    include Mandate
+
+    def call
+      env = ENV["EXERCISM_ENV"] || ENV["RAILS_ENV"] || ENV["APP_ENV"]
+      raise Exercism::ConfigError, "No environment set - set one of EXERCISM_ENV, RAILS_ENV or APP_ENV" unless env
+
+      unless %w[development test production].include?(env)
+        raise Exercism::ConfigError, "environments must be one of development test production. Got #{env}"
```

Comment:
```suggestion
        raise Exercism::ConfigError, "environment must be one of development, test or production. Got #{env}."
```

This more closely matches the other message, I think.  And fixes the plurality.

Reply:
Thanks :)